{
  "debianbug": 931316,
  "description": "An issue was discovered in Django 1.11 before 1.11.22, 2.1 before 2.1.10, and 2.2 before 2.2.3. An HTTP request is not redirected to HTTPS when the SECURE_PROXY_SSL_HEADER and SECURE_SSL_REDIRECT settings are used, and the proxy connects to Django via HTTPS. In other words, django.http.HttpRequest.scheme has incorrect behavior when a client uses HTTP.",
  "releases": {
    "bullseye": {
      "fixed_version": "1:1.11.22-1",
      "repositories": {
        "bullseye": "1:1.11.23-1~deb10u1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "1:1.11.22-1~deb10u1",
      "repositories": {
        "buster": "1:1.11.21-1",
        "buster-security": "1:1.11.23-1~deb10u1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "1.7.11-1+deb8u6",
      "repositories": {
        "jessie": "1.7.11-1+deb8u3",
        "jessie-security": "1.7.11-1+deb8u7"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "1:1.11.22-1",
      "repositories": {
        "sid": "2:2.2.4-1"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "1:1.10.7-2+deb9u5",
      "repositories": {
        "stretch": "1:1.10.7-2+deb9u4",
        "stretch-security": "1:1.10.7-2+deb9u6"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}